fs

```diff
@@ -24,10 +24,11 @@ def cli(ctx, fleetctl_endpoint, timeout):
 @click.argument('service-name', type=str)
 @click.argument('unit-file', type=click.Path(exists=True))
 @click.option('--count', type=int, default=3)
+@click.option('--restart', type=click.Choice(['always', 'no']), default=None, help='Restart strategy for unit. If not specified, it uses what is defined in the unit.')
```

Comment:
> Don't you want to add all restart options? (no, on-success, on-failure, on-abnormal, on-watchdog, on-abort, or always)
Not really... I find them overly complex and unnecessary 😄 In practice I think `no` and `always` should be enough. And I don't think we have used any of the other statuses anywhere.